docs/faq.md

```diff
@@ -114,3 +114,18 @@ For the Example project, run the `EarlGreyExampleSwiftTests` target once then fi
 For physical device builds, replace `Debug-iphonesimulator` with `Debug-iphoneos`.
 
 You can verify the `EarlGrey.framework` is included in the bundle at `EarlGreyExampleSwiftTests.xctest/Frameworks/EarlGrey.framework`
+
+**How should I handle animations?**
+
+Disabling animations improves test performance and eliminates a source of flakiness.
```

Comment:
The same argument can be made for Android and I don't see anyone advocating speeding up animations there. Instead the advice, consistently, has been to [disable animations](https://youtu.be/isihPOY2vS4?t=1302). That's true both for system animations and app specific animations. To the extent EarlGrey is Espresso for iOS, it'd be nice to not have a completely opposite position.

I'll update the FAQ so the pros/cons are more obvious, specifically the trade off between more coverage via animations and the resulting flakiness.